src/test/java/homework23/ui/utils/HelpFunc.java

```diff
@@ -0,0 +1,82 @@
+package homework23.ui.utils;
+
+import com.codeborne.selenide.SelenideElement;
+import com.github.javafaker.Faker;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+import static com.codeborne.selenide.Selenide.$;
+import static com.codeborne.selenide.Selenide.$x;
+
+public class HelpFunc {
+    private static final SelenideElement titleInCartEl = $("td>a");
+    private static final SelenideElement titleCheckOutEl = $("td>a");
+    private static final SelenideElement priceInCartEl = $x("//a/../following-sibling::td");
+    private static final SelenideElement priceCheckOutEl = $x("//a/../following-sibling::td/following-sibling::td");
+    private static final SelenideElement quantityInCartEl = $("#mat-badge-content-0");
+    private static final SelenideElement quantityCheckOutEl = $x("//a/../following-sibling::td");
+    private static final SelenideElement orderIdEl = $x("//tr/td");
+    private static final SelenideElement orderOnEl = $x("//tbody/tr/td/following-sibling::td");
+    private static final SelenideElement orderTotalEl = $x("//tbody/tr/td/following-sibling::td/following-sibling::td");
```

Comment:
these elements and methods where they are used should be moved to proper page object classes